### PR TITLE
Fixes padding on server card, hopefully better responsiveness too

### PR DESCRIPTION
**Pull request by Lunaversitay**
Apparently on laptop screens that had certain things such as a sidebar from the browser (e.g. from opera gx) breakpoints wouldn't work properly. This should hopefully fix that issue.

imo it'd just be better for laptops to see the entire card in 1 column instead of 2 to prevent any weird alignments - on desktop screens it should still display 2 columns however.

Also added a tad bit of padding to the server card icons for now but I believe there is a bigger underlying issue as these issues do not appear in local builds.

Laptop Preview (1280x800):

### CODE REVIEW

**Comment by Lunaversitay** [comment]
Should address #5 & #150 and a portion of #156 for the "My servers" page

File: src/views/ServerCard.vue [hunk 1]
```diff
@@ -16,7 +16,7 @@
                             <router-link :to="{name: 'server.system.index', params: { server: server?.uuidShort }}">
                                 {{ server?.name }}
                             </router-link>
-                            <span class="text-white/50 block md:inline text-sm leading-none tracking-tight pl-2" v-clipboard>
+                            <span class="text-white/50 hidden md:block md:inline text-sm leading-none tracking-tight pl-2" v-clipboard>
```
Comment (victiondev) [inline src/views/ServerCard.vue:19]: md: block md:inline cancel eachother out - use either

File: src/views/ServerCard.vue [hunk 2]
```diff
@@ -34,7 +34,7 @@
 
                 <button
                     v-tippy="'generic.server.go_to_console'"
-                    class="border border-primary-50 px-2 rounded text-primary-0 hover:text-white mr-4 hidden md:block"
+                    class="border border-primary-50 px-2 py-1 rounded text-primary-0 hover:text-white mr-4 hidden md:block"
```
Comment (victiondev) [inline src/views/ServerCard.vue:37]: These buttons should be slightly rectangular, not squares

File: src/views/ServerCard.vue [hunk 3]
```diff
@@ -43,7 +43,7 @@
                     <router-link :to="{name: 'server.system.index', params: { server: server?.uuidShort }}">
                         <button
                             v-tippy="'generic.server.manage'"
-                            class="border border-primary-50 px-2 rounded text-primary-0 hover:text-white hidden md:block"
+                            class="border border-primary-50 px-2 py-1 rounded text-primary-0 hover:text-white hidden md:block"
```
Comment (victiondev) [inline src/views/ServerCard.vue:46]: These buttons should be slightly rectangular, not squares

**Review by victiondev — CHANGES_REQUESTED** [review]

File: src/views/ServerCard.vue (hunk 2 above)
Comment (Lunaversitay) [reply src/views/ServerCard.vue:37]: They'll look rectangular in production due to whatever bug is causing them not to mirror local

Here's a preview of it in prod:

File: src/views/ServerCard.vue (hunk 1 above)
Comment (Lunaversitay) [reply src/views/ServerCard.vue]: oops, fixed in https://github.com/wisp-gg/frontend/pull/193/commits/c8ca762f177fb0cc0f12415c87f00c40e3a3c9cc

File: src/views/ServerCard.vue (hunk 2 above)
Comment (Lunaversitay) [reply src/views/ServerCard.vue]: removed padding in https://github.com/wisp-gg/frontend/pull/193/commits/c8ca762f177fb0cc0f12415c87f00c40e3a3c9cc so we can fix the underlying issue with the icons causing scaling issues on prod

File: src/views/ServerCard.vue (hunk 3 above)
Comment (Lunaversitay) [reply src/views/ServerCard.vue]: removed padding in https://github.com/wisp-gg/frontend/pull/193/commits/c8ca762f177fb0cc0f12415c87f00c40e3a3c9cc so we can fix the underlying issue with the icons causing scaling issues on prod

**Pull request merged**